### DISCUSSION TITLE
Flashed JeyersUI to GD chip

### DISCUSSION THREAD

**Discussion by Menethoran** [Jyers/Marlin#2036] [Q&A]
Hi, I just joined the ender discord community and was asking around about print adhesion only to stumble across the fact that I accidently was capable of flashing JeyersUI onto an ender 3 v2 with 4.2.2 mobo and GD chipset.

I thought I'd mention it to you.

I do have a second machine identical to the first that is still stock. I can attempt to flash it as well to see if I can replicate results.

**Comment by MateoBarbato** [comment]
Hey man! Congrats, can i ask you how did you do it? Did you install a compiled version for a stm chip? 
Im going to try when i finish a print and i'll see how it goes
